examples/ensemble/plot_monotonic_constraints.py

```diff
@@ -28,7 +28,7 @@
 
 rng = np.random.RandomState(0)
 
-n_samples = 5000
+n_samples = 1000
```

Comment:
I reduce the number of samples to make the plot less crowded while conveying the same intuitions and furthermore making the example run faster.